core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/RepairStateImpl.java

```diff
@@ -194,10 +192,9 @@ private long partiallyRepairedTableRepairedAt(final long maxRepairedAt, final Re
         {
             next -= old.getEstimatedRepairTime();
         }
-        if (LOG.isInfoEnabled())
+        if (LOG.isDebugEnabled())
         {
-            LOG.info("Table {} has been partially repaired, next repair {}",
-                    myTableReference,
+            LOG.debug("Table {} has been partially repaired, next repair at/after {}", myTableReference,
```

Comment:
I'm a sucker for consistency. The other method includes "repaired at" timestamp, why not have it here as well?